scripts/webpack/webpack.dev.js

```diff
@@ -35,6 +35,18 @@ module.exports = (env = {}) => {
       ignored: /node_modules/,
     },
 
+    resolve: {
+      alias: {
```

Comment:
Can you confirm if this will remove the `alias` that we get from common? Reading the docs about this [merge](https://www.npmjs.com/package/webpack-merge#mergeconfiguration--configuration) function makes me believe it will.

Reply:
I tested this before and I can confirm it's fine! 

![image](https://github.com/grafana/grafana/assets/46142/05249190-25ff-478d-a160-1881f7c67dd6)